omnibus/config/software/pip2.rb

```diff
@@ -24,4 +24,10 @@
   end
 
   command "#{python_bin} setup.py install --prefix=#{python_prefix}"
+
+  if ohai["platform"] != "windows"
+    block do
+      FileUtils.rm_f(Dir.glob("#{install_dir}/embedded/lib/python2.7/site-packages/pip-19.0.3-py2.7.egg/pip/_vendor/distlib/*.exe"))
```

Comment:
Should we use `pip-*-py2.7.egg`? Otherwise, we might forget to change this if/when upgrading pip.